web_src/css/modules/button.css

```diff
@@ -85,6 +85,13 @@ It needs some tricks to tweak the left/right borders with active state */
   box-shadow: none;
 }
 
+/* apply the vertical padding of .compact to non-compact buttons when they contain a svg as they
+   would otherwise appear too large. Seen on "RSS Feed" button on repo releases tab. */
+.ui.small.button:not(.compact):has(.svg) {
```

Comment:
`has` doesn't work for firefox.

And since the button has "flex align center" layout, maybe the padding could be 0

Reply:
`has` works since Firefox 120, and even if the browser does not support it, it's graceful degration.

https://developer.mozilla.org/en-US/docs/Web/CSS/:has

Not sure what you mean with 0 padding. All buttons have padding from fomantic.

Reply:
The problem is fundamentally in the button design. The height of the button depends on the content height which works well for single-line text, but breaks if anything larger is in there. I wonder how other frameworks solve this problem without adding "hacks" that specifically target icons in buttons.